README.md

```diff
@@ -1,13 +1,17 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
+Firts node.js project assigment from Technigo 
+Create my own API
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Its a lot of new things to learn and I find it difficult. 
+Got some help from former students and re watching the lectures.
+
+Got stuck trying to make a query for the "title", cant make it work. I commented out the code but will return to have an other look when I have more time.
```

Comment:
It would be very nice to have some instructions, how to use your API. Remember to make documentation on what endpoints and queries your user can use. 🤓